backend/test/integrated/teamspace.js

```diff
@@ -366,4 +373,304 @@ describe("Checking Quota Info  ", function() {
 		});
 	});
 
+	const defaultRiskCategories = [
+		{ "value" : "commercial_issue", "label" : "Commercial Issue" },
+		{ "value" : "environmental_issue", "label" : "Environmental Issue" },
+		{ "value" : "health_-_material_effect", "label" : "Health - Material effect" },
+		{ "value" : "health_-_mechanical_effect", "label" : "Health - Mechanical effect" },
+		{ "value" : "safety_issue_-_fall", "label" : "Safety Issue - Fall" },
+		{ "value" : "safety_issue_-_trapped", "label" : "Safety Issue - Trapped" },
+		{ "value" : "safety_issue_-_event", "label" : "Safety Issue - Event" },
+		{ "value" : "safety_issue_-_handling", "label" : "Safety Issue - Handling" },
+		{ "value" : "safety_issue_-_struck", "label" : "Safety Issue - Struck" },
+		{ "value" : "safety_issue_-_public", "label" : "Safety Issue - Public" },
+		{ "value" : "social_issue", "label" : "Social Issue" },
+		{ "value" : "other_issue", "label" : "Other Issue" },
+		{ "value" : "unknown", "label" : "UNKNOWN" }
+	];
+	const defaultRiskCategoryLabels = defaultRiskCategories.map(x => x.label);
+	const defaultTopicTypes =  [
+		{ "value" : "for_information", "label" : "For information" },
+		{ "value" : "vr", "label" : "VR" }
+	];
+	const defaultTopicTypeLabels = defaultTopicTypes.map(x => x.label);
+
+	describe("Update teamspace settings", function(done) {
+		const user =  imsharedTeamspace;
+		const newRiskCategories = [
+			{ "value": "new_cat_1", "label": "New Cat 1" },
+			{ "value": "new_cat_2", "label": "New Cat 2" }
+		];
+		const newRiskCategoryLabels = newRiskCategories.map(x => x.label);
+		const newTopicTypes = [
+			{ "value": "new_type_1", "label": "New Type 1" },
+			{ "value": "new_type_2", "label": "New Type 2" }
+		];
+		const newTopicTypeLabels = newTopicTypes.map(x => x.label);
+
+		before(function(done) {
+			this.timeout(timeout);
+			agent.post("/login")
+				.send({username: user.user, password: user.password})
+				.expect(200, done);
+
+		});
+
+		it("set defaults should succeed", function(done) {
```

Comment:
again here:

- if username doesn't exist
- If user is not part of teamspace
- if user is not admin